packages/grafana-ui/src/components/BigValue/BigValue.tsx

```diff
@@ -43,6 +54,7 @@ export interface Props extends Themeable {
   graphMode: BigValueGraphMode;
   justifyMode?: BigValueJustifyMode;
   alignmentFactors?: DisplayValueAlignmentFactors;
+  nameAndValue?: BigValueNameAndValueOption;
```

Comment:
```suggestion
  showText?: BigValueTextOption;
```

Reply:
changed to textMode and BigValueTextMode